src/lib/ngsi/Request.cpp

```diff
@@ -75,6 +75,7 @@ const char* requestType(RequestType rt)
   case Ngsi10SubscriptionsConvOp:                   return "Ngsi10SubscriptionsConvOp";
 
   case LogRequest:                                  return "Log";
+  case LogLevelRequest:                             return "LogLevel";
   case VersionRequest:                              return "Version";
```

Comment:
LogLevelRequest is the new request.

LogRequest are the old ones:

/v1/log/traces   et al